src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    private static final Random random = new Random();
+    private final Color[] color = Color.values();
+
+    public static String getRandomColor() {
+        int index = random.nextInt(Color.values().length);
+        return Color.values()[index].name().toLowerCase();
```

Comment:
Don't use `toLowerCase()` in the `getRandomColor()` method. This could lead to unexpected behavior if the enum names are used for logic elsewhere in the code, as enums are typically expected to be in uppercase.